virttest/utils_net.py

```diff
@@ -2553,7 +2553,7 @@ class VirtIface(propcan.PropCan, object):
     def __getstate__(self):
         state = {}
         # pylint: disable=E1133
-        for key in self.__class__.__all_slots__:
+        for key in self.__class__.__all_slots__: # pylint: disable=E1133
```

Comment:
already fixed on Line 2555.

Reply:
and I remember @dzhengfy has solved these problems, so do we still need this PR?

Reply:
@Yingshun 
there is tricky here.It looks like    # pylint: disable=E1133 before line doesn't work. But appending end of line works instead

Reply:
@Yingshun 
I didn't notice there is already fixed PR, maybe close it.

Reply:
It was fixed by #3140 . And I saw that ci checking for  #3137 was passed, so it's working well I think, right?